process/check.py

```diff
@@ -449,3 +455,22 @@ def _process_service_check(self, name, nb_procs, bounds, tags):
             tags=service_check_tags,
             message=message_str % (status_str[status], nb_procs, name)
         )
+
+    def _filter_by_user(self, user, pids):
+        """
+        Filter pids by it's username.
+        :param user: string with name of system user
+        :param pids: set of pids to filter
+        :return: set of filtered pids
+        """
+        filtered_pids = set()
+        for pid in pids:
+            try:
+                proc = psutil.Process(pid)
+                if proc.username() == user:
+                    self.log.debug("Collecting pid %s belonging to %s", pid, user)
```

Comment:
Would it be helpful to log something similar to this when we skip a file as well?